data/weapons-camo.ts

```diff
@@ -296,11 +589,41 @@ export const MELEE: Weapon[] = [
     name: 'KNIFE',
     type: WeaponType.MELEE,
     camo: {
-      fromBehind: 30,
-      crouching: 30,
+      kill: 50,
+      behind: 30,
+      crouch: 50,
+      noDeath2: 30,
       deadSilence: 50,
-      kills2NoDie: 30,
-      kills3NoDie: 30,
+      noDeath3: 30,
+    },
+  },
+];
+
+export const LAUNCHER: Weapon[] = [
+  {
+    id: 'fhj18',
+    name: 'FHJ-18',
+    type: WeaponType.LAUNCHER,
+    camo: {
+      streaks: 50,
+      UAV: 30,
```

Comment:
I would change this to uav. The javascript naming convention is to use camelCase to indicate a variable. The same with VTol, it should be just lowercase vtol.

Reply:
VSCode must have thrown a warning since this fhj-18 doesn't have the kill field. Great stuff with changing it to optional.

Reply:
Vscode helped me with finding the error, it's nice knowing the errors while typing the code

Reply:
I'll change UAV and VTol to lowercase

Reply:
I know right, helps in learning for sure.